### DISCUSSION TITLE
Why I need Wi-Fi?

### DISCUSSION THREAD

**Discussion by vincvinc563** [SideStore/SideStore#380] [General]
Hello, I know the app is still under development but I was wondering why it wasn't possible to refresh using only wifi and not 4g.

**Comment by RandomModderJDK** [comment]
Because SideStore emulates an Apple Laptop with the VPN. The VPN lets you iDevice think that you are on the same network with an Apple laptop running xcode. Unfortunately iDevices do not support running Developers actions trough cellular. How could an ordinary laptop connect to an iDevice through cellular?

**Reply by vincvinc563** [reply]
oh okay, I didn't know about this! thank you for explaining it to me. have a Good day

**Comment by amsam0** [comment]
lockdownd refuses to do anything if you're on cellular (it works on a hotspot though)
